open-vm-tools-centos/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos 
+
+
+
+LABEL summary="The open-vm-tools guest agent" \
+      io.k8s.description="The open-vm-tools agent is providing information about the virtual machine and allows to restart / shutdown the machine via VMware products. This image is intended to be used with virtual machines running Centos Atomic Host." \
+      io.k8s.display-name="open-vm-tools guest agent" \
+      architecture="x86_64" \
+      BZComponent="open-vm-tools" \
+      maintainer="davis phillips <dphillip@redhat.com>"
+
+ENV SYSTEMD_IGNORE_CHROOT=1
+
+RUN yum -y --disablerepo=\* --enablerepo=rhel-7-server-rpms install yum-utils && \
+  yum-config-manager --disable \* && \
+  yum-config-manager --enable rhel-7-server-rpms && \
+  yum clean all
```

Comment:
Are these 4 lines necessary? The container seems to build fine without specifically enabling rhel-7-server-rpms, and I get: `Error getting repository data for rhel-7-server-rpms, repository not found` if I leave these in.